<ui version="4.0" >
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>311</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Configuration Dialog</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>USARSim Connection</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Host</string>
        </property>
        <property name="buddy" >
         <cstring>hostEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="hostEdit" />
      </item>
      <item>
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Port</string>
        </property>
        <property name="buddy" >
         <cstring>portEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="portEdit" >
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>65535</number>
        </property>
        <property name="value" >
         <number>7435</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Signal Strength Computation Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Cutoff</string>
        </property>
        <property name="buddy" >
         <cstring>cutoffEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QDoubleSpinBox" name="cutoffEdit" >
        <property name="decimals" >
         <number>3</number>
        </property>
        <property name="minimum" >
         <double>-200.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>0.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
        <property name="value" >
         <double>-93.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>N</string>
        </property>
        <property name="buddy" >
         <cstring>eNEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QDoubleSpinBox" name="eNEdit" >
        <property name="decimals" >
         <number>3</number>
        </property>
        <property name="minimum" >
         <double>0.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
        <property name="value" >
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Pdo</string>
        </property>
        <property name="buddy" >
         <cstring>ePdoEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QDoubleSpinBox" name="ePdoEdit" >
        <property name="decimals" >
         <number>3</number>
        </property>
        <property name="minimum" >
         <double>-200.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>0.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
        <property name="value" >
         <double>-46.670000000000002</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Do</string>
        </property>
        <property name="buddy" >
         <cstring>eDoEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QDoubleSpinBox" name="eDoEdit" >
        <property name="decimals" >
         <number>3</number>
        </property>
        <property name="minimum" >
         <double>0.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
        <property name="value" >
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Easy Parameter Setting by Cutoff Distance</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" >
      <item>
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>Cutoff in Meters</string>
        </property>
        <property name="buddy" >
         <cstring>distance</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="distance" >
        <property name="suffix" >
         <string>m</string>
        </property>
        <property name="maximum" >
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>hostEdit</tabstop>
  <tabstop>portEdit</tabstop>
  <tabstop>cutoffEdit</tabstop>
  <tabstop>ePdoEdit</tabstop>
  <tabstop>eNEdit</tabstop>
  <tabstop>eDoEdit</tabstop>
  <tabstop>distance</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>257</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>325</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cutoffEdit</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ConfigDialog</receiver>
   <slot>recomputeMeterCutoff()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>177</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel" >
     <x>375</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ePdoEdit</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ConfigDialog</receiver>
   <slot>recomputeMeterCutoff()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>155</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel" >
     <x>377</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eNEdit</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ConfigDialog</receiver>
   <slot>recomputeMeterCutoff()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>321</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel" >
     <x>375</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eDoEdit</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ConfigDialog</receiver>
   <slot>recomputeMeterCutoff()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>310</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel" >
     <x>372</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>distance</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ConfigDialog</receiver>
   <slot>recomputeParameters()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>295</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel" >
     <x>385</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>recomputeMeterCutoff()</slot>
  <slot>recomputeParameters()</slot>
 </slots>
</ui>
